### DISCUSSION TITLE
Should this build option be deprecated

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/fsharp#13755] [General]
# prefer .net framework version of the compiler when available
![image](https://user-images.githubusercontent.com/35312807/186112055-9e330728-e707-4162-8da4-76f285d151a7.png)
On Visual Studio 2022 preview with fsharp library project targetting .NET 6

**Comment by vzarytovskii** [comment]
No, not yet, it was deliberately added last year, we will deprecate it eventually though.
